src/test/java/core/basesyntax/HelloWorldTest1.java

```diff
@@ -0,0 +1,183 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.CustomeException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class HelloWorldTest1 {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userWithNullLogin_NotOk() {
+        User user = new User();
+        user.setLogin(null);
+        user.setPassword("12345678");
+        user.setAge(22);
+        assertThrows(CustomeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullPassword_NotOk() {
+        User user = new User();
+        user.setLogin("user12345");
+        user.setPassword(null);
+        user.setAge(41);
+        assertThrows(CustomeException.class, () -> {
+            registrationService.register(user);
+        });
+    }
+
+    @Test
+    void nullAge_NotOk() {
+        User user = new User();
+        user.setAge(null);
+        Exception exception = assertThrows(CustomeException.class, () ->
+                registrationService.register(user));
+        String excepted = "User login,password or age is null";
```

Comment:
The expected message 'User login,password or age is null' does not match the actual message format. Ensure the expected message matches the actual exception message.